initproducerid_test.go

```diff
@@ -31,7 +32,7 @@ func TestClientInitProducerId(t *testing.T) {
 	}
 
 	// Now establish a connection with the transaction coordinator
-	transactionCoordinator := TCP(fmt.Sprintf("%s:%d", respc.Coordinator.Host, respc.Coordinator.Port))
+	transactionCoordinator := TCP(net.JoinHostPort(respc.Coordinator.Host, fmt.Sprintf("%d", respc.Coordinator.Port)))
```

Comment:
Same as above about `strconv.Itoa`
```suggestion
	transactionCoordinator := TCP(net.JoinHostPort(respc.Coordinator.Host, strconv.Itoa(respc.Coordinator.Port)))
```